Mathlib/Init/Data/Int/Basic.lean

```diff
@@ -2,13 +2,17 @@
 Copyright (c) 2016 Jeremy Avigad. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Jeremy Avigad
-
-The integers, with addition, multiplication, and subtraction.
 -/
 import Mathlib.Mathport.Rename
 import Mathlib.Init.Data.Nat.Notation
 import Std.Data.Int.Order
 
+/-!
+# Integers
+
+This file contains basic properties of integers with addition, multiplication, and subtraction.
```

Comment:
Can you make this more specific?
- this file defines `natMod`; this could be mentioned
- is "contains basic lemmas about addition, multiplication and subtraction of integers" a better phrasing?

Besides, I wonder if @YaelDillies would like to eliminate this file as well. In this case, it might be more useful to do that instead. :-)

Reply:
Yep indeed this file is going away soon

Reply:
Ok, goof to know.